asciidoc/volume3/tf3-ch-8.3.2.13-metric-display-precision.adoc

```diff
@@ -0,0 +1,96 @@
+
+In this case, the *@StepWidth* of the *pm:Range* element is the same as the metric's *@Resolution* in order to preserve the resolution which would be superseded by a different *pm:Range/@StepWidth* otherwise.
+
+.Ventilator Minute Volume Parameter Representation
+====
+A ventilator measures the *Minute Volume (MV)* in *L/min* with a resolution of one decimal place, but dependent on the value, the number is either displayed with no or one decimal place.
+The <<actor_somds_provider>> defines the following in the *pm:NumericMetricDescriptor* of the *MV* metric:
+
+* *@Resolution* = "0.1"
+* *pm:TechnicalRange[0]/@Upper* = "100.0"
+* *pm:TechnicalRange[0]/@Lower* = "10.0"
+* *pm:TechnicalRange[0]/@StepWidth* = "0.1"
+* *pm:TechnicalRange[0]/ext:Extension/mpkp:DisplayPrecision/@StepWidth* = "1"
+* *pm:TechnicalRange[1]/@Upper* = "9.9"
+* *pm:TechnicalRange[1]/@Lower* = "0.0"
+* *pm:TechnicalRange[1]/@StepWidth* = "0.1"
+
+The first range is defined from *10.0 L/min* to  *100.0 L/min*.
+The measurement precision stays the same (*@Resolution* and *@StepWidth* are identical).
+However, the <<actor_somds_provider>> indicates to the <<actor_somds_consumer>> that the *MV* parameter value needs to be displayed with no decimal places (*mpkp:DisplayPrecision/@StepWidth* = "1").
+
+The second range is defined from *9.9 L/min* to  *0.0 L/min*.
+Since there is no specific display precision defined, the <<actor_somds_consumer>> knows that the *MV* parameter value needs to be displayed with one decimal places (*@StepWidth* = "0.1").
+
+It would also be correct to omit the second range element, since the implied display precision equals the metric's *@Resolution* in this case.
+====
+
+====== Requirements
+
+.R1015
+[sdpi_requirement#r1015,sdpi_req_level=shall]
+****
+If the display precision differs from the metric's *@Resolution* for a specific metric's *@Value* or *@Samples* range where no *pm:TechnicalRange* or *pm:PhysiologicalRange* element is defined, the <<actor_somds_provider>> shall define a descriptor *pm:TechnicalRange* element that covers this metric's *@Value* or *@Samples* range, and where the *@StepWidth* equals the *@Resolution* and the *ext:Extension/mpkp:DisplayPrecision/@StepWidth* defines the display precision.
+
+.Notes
+[%collapsible]
+====
+NOTE: The <<actor_somds_provider>> could also define *pm:PhysiologicalRange* element in the metric state in lieu of *pm:TechnicalRange* element in the metric descriptor.
+However, it is recommended to define the appropriate *pm:Range* element in the descriptor.
+====
+****
+
+.R1016
+[sdpi_requirement#r1016,sdpi_req_level=shall]
+****
+A <<actor_somds_consumer>> shall determine the _active range_ for the current metric's *@Value* or *@Samples* by the following priority order:
+
+. The *pm:PhysiologicalRange* element of the metric state where the current metric's *@Value* or *@Samples* falls into.

```

Comment:
`@Samples` is an array of values, meaning it can have values which fall into many different Ranges. The definition of applicability needs to be done over every single value in said array.

Reply:
Reworked serveral sections and R1016. It is hard to give a clear guidance since this depends on the SDC consumer application.